File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        res = []
        count = {}
        for i in nums:
            if not i in count:
                count[i] = 1
            else:
                count[i] += 1
        return [x[0] for x in sorted(count.items(), key=lambda x: x[1], reverse=True)[:k]]

File: Data Structures & Algorithms/contains-duplicate-ii/submission-0.cs
public class Solution {
    public bool ContainsNearbyDuplicate(int[] nums, int k) {
        for (int i = 0; i < nums.Length; i++) {
            if (i == nums.Length - 1) continue; 
            int j = i + 1;
            while (j - i <= k && j < nums.Length) {
                if (nums[i] == nums[j]) return true;
                j++;
            }
        }
        return false;
    }
}